Taskfile.yaml

```diff
@@ -201,7 +201,7 @@ tasks:
       - featurebyte/**/*
       - tests/**/*
     cmds:
-      - poetry run pytest --timeout=240 --junitxml=pytest.xml.0 -n auto --cov=featurebyte tests/unit
+      - poetry run pytest --timeout=240 --junitxml=pytest.xml.0 -n auto --cov=featurebyte tests/unit tests/unit/conftest.py
 
```

Comment:
just wondering why you have to specify the conftest directly. Isn't the previous `tests/unit` already specified the conftest?

Reply:
If we don't specify the conftest, the test won't run. That's the reason why the issue is not caught. Another option is to move the test to generate the fixtures into a file starts with `test_*`.